pkg/issuer/acme/dns/route53/route53.go

```diff
@@ -245,16 +245,23 @@ func (r *DNSProvider) getHostedZoneID(fqdn string) (string, error) {
 		return "", err
 	}
 
-	var hostedZoneID string
+	var zoneToID = make(map[string]string)
```

Comment:
```suggestion
	zoneToID : = make(map[string]string)
```

That looks a bit more Go-native :)

Reply:
ack.